invariant/stdlib/invariant/detectors/code.py

```diff
@@ -5,8 +5,7 @@
 CODE_SHIELD_DETECTOR = None
 SEMGREP_DETECTOR = None
 
-@cache
```

Comment:
Any reason why cache was disabled here?

Reply:
I think I missed this in the merge. Should not be disabled. Will fix when amending.